ush/wave_outp_cat.sh

```diff
@@ -136,8 +136,8 @@
     echo '*************************************************** '
     echo ' '
     set_trace
-    err=2; export err;${errchk}
-    exit $err
+    export err=2
+    err_chk
```

Comment:
Add fatal error message from line 135 to `err_chk` and clean up echos in if-block?

Reply:
Yes, moved.